### PR TITLE
Integration with antd

### PR DESCRIPTION
**Pull request by ri7nz**
**What**: Integration with antd and moved from here https://github.com/devpendent/website/pull/24

**Why**: (replace react-bulma-components https://github.com/devpendent/website/issues/1)

**How**: we can use antd component like this
```
import { Button } from 'antd'
```

**Checklist**:

- [ ] Tests
- [ ] Ready to be merged

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -5,13 +5,3 @@
  */
 
 // You can delete this file if you're not using it
-const path = require('path')
-
-exports.onCreateWebpackConfig = ({ actions, loaders, plugins, stage }) => {
-  actions.setWebpackConfig({
-    resolve: {
-      modules: ['src', 'node_modules'],
```
Comment (zainfathoni) [inline gatsby-node.js]: We still need this `modules` here, no wonder the build failed in Netlify. 😅

File: gatsby-node.js [hunk 2]
```diff
@@ -5,13 +5,3 @@
  */
 
 // You can delete this file if you're not using it
-const path = require('path')
```
Comment (zainfathoni) [inline gatsby-node.js]: This still needs to be removed.

File: gatsby-node.js (hunk 1 above)
Comment (ri7nz) [reply gatsby-node.js]: I'm sorry, my bad @zainfathoni  👍

File: gatsby-node.js [hunk 3]
```diff
@@ -11,12 +11,7 @@ exports.onCreateWebpackConfig = ({ actions, loaders, plugins, stage }) => {
   actions.setWebpackConfig({
     resolve: {
       modules: ['src', 'node_modules'],
-      alias: {
-        '_variables.sass': path.resolve(
-          __dirname,
-          './src/styles/_variables.sass'
-        )
-      }
+      alias: {}
```
Comment (zainfathoni) [inline gatsby-node.js]:
```suggestion
```

**Pull request merged**